caffeinated/admin/extend/registration_form/Extend_Registration_Form_Admin_Page.core.php

```diff
@@ -1462,14 +1463,15 @@ private function _verify_pcre_support(EE_Registration_Config $EE_Registration_Co
 
     public function setSessionLifespan()
     {
-        $session_lifespan_form = new SessionLifespanForm();
+        $session_lifespan_form = new SessionLifespanForm(new SessionLifespanOption());
```

Comment:
if we use the loader, then any dependencies can be automatically injected

```php
$session_lifespan_form = $this->loader->getNew(SessionLifespanForm::class);
```

`$this->loader` should be available within all admin pages

Reply:
Done.